src/sources/http_server.rs

```diff
@@ -130,7 +130,7 @@ pub struct SimpleHttpConfig {
     #[serde(default = "default_http_method")]
     method: HttpMethod,
 
-    /// Specifies the HTTP response status code on request success.
+    /// Specifies the HTTP response status code that will be returned on successful requests.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Note that with any changes to the config field doc comments, you'll want to run `make generate-component-docs` to re-generate the cue files, and then commit the changes.‏‏